akka-docs/src/test/scala/docs/future/FutureDocSpec.scala

```diff
@@ -479,31 +482,43 @@ class FutureDocSpec extends AkkaSpec {
   }
 
   "demonstrate usage of pattern.after" in {
+    import akka.actor.typed.scaladsl.adapter.ClassicActorSystemOps
+    val system: ActorSystem[Nothing] = this.system.toTyped
     //#after
-    // TODO after is unfortunately shadowed by ScalaTest, fix as part of #3759
-    // import akka.pattern.after
+    import akka.actor.typed.scaladsl.adapter._
+    import akka.pattern.after
+
+    implicit val scheduler: akka.actor.Scheduler = system.scheduler.toClassic
+    implicit val ec: ExecutionContext = system.executionContext
```

Comment:
we could see if it would be possible to support an implicit ActorSystem (ClassicActorSystemProvider) as an alternative to scheduler and ec

Reply:
I did that now, but there are loads of overloads already...